src/services/CommandsService.js

```
@@ -0,0 +1,28 @@
+
+export async function getAllCommands(page = 0, size = 10) {
+    try {
+        const response = await axios.get(`${API_URL}/${URL_COMMAND}/all`, {
+            params: {
+                page: page,
+                size: size
+            }
+        });
+        return response.data;
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)